crypto-primitives/src/sponge/mod.rs

```diff
@@ -17,6 +17,11 @@ pub use absorb::*;
 /// [cos]: https://eprint.iacr.org/2019/1076
 pub mod poseidon;
 
+/// The sponge for Merlin
+///
+///
```

Comment:
Is this perhaps missing a link/description?

Reply:
@autquis

Reply:
Sorry, it slipped out of my hands!